plugin/src/makeWorklet.ts

```diff
@@ -370,7 +372,7 @@ export function makeWorklet(
     functionName,
     transformed.map
   );
-  if (!funString) throw new Error("'funString' is not defined\n");
+  assertIsDefined(funString);
```

Comment:
In which case `funString` is not defined? Is it possible to use TypeScript to ensure that it's defined?